wire-ios/Wire-iOS/Sources/UserInterface/Components/MessagePreviewView.swift

```diff
@@ -144,7 +145,15 @@ final class MessageThumbnailPreviewView: UIView {
 
     private func editIcon() -> NSAttributedString {
         if message.updatedAt != nil {
-            return "  " + NSAttributedString(attachment: NSTextAttachment.textAttachment(for: .pencil, with: iconColor, iconSize: 8))
+            return NSAttributedString(
+                attachment: NSTextAttachment.textAttachment(for: .pencil,
+                with: iconColor,
```

Comment:
is that intended formatting?

Reply:
I think so, otherwise the line would be too long.

Reply:
I've been used to keep indentation of the same level arguments, in your case it's mixed

I would suggest something like this for this initializer: 
```swift
return NSAttributedString(
  attachment: NSTextAttachment.textAttachment(
    for: .pencil,
    with: iconColor,
    iconSize: 8,
    insets: UIEdgeInsets(
      top: 0,
      left: leftEditIconInset,
      bottom: 0,
      right: 0
    )
  )
)
```

Reply:
Done!